schema.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS imatable;
+
```

Comment:
Looks good so far! Might want to rename the table to something more semantic though!